content/en/account_management/audit_logs.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Audit Logs
+kind: documentation
+further_reading:
+- link: "/account_management/org_settings/"
+  tag: "Documentation"
+  text: "Learn more about organization settings"
+---
+
+<div class="alert alert-warning">This feature is in public beta. Contact <a href="https://www.datadoghq.com/support/">Datadog support</a> for more information.</div>
+
+## Overview
+
+As an administrator or security team member, you can see who is using Datadog within your organization and the context in which they are using Datadog with [Audit Logs][1]. As an individual, you can see a stream of your own actions, too.
+
+There are two types of events that can occur within an audit log:
+
+1. Request events, which translate all requests made to Datadog’s API into a customer record
+2. Product-specific events, such as:
+    - What has changed within your organization or account. For example, if a user changes a dashboard, you are able to see who made the last change and at what time.
+    - Who accessed and viewed certain information within your environment, for compliance audits.
+    - A breaking event, such as seeing what API calls led up to the event.
+    - An audit trail of information relating to user access. For example, if a user receives a policy denied warning.
+
+## Setup
+
+Audit Logs is self-service enabled. To enable, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Click the **Enable** button.
+
+{{< img src="account_management/audit_logs/setup.png" alt="Audit Logs setup in Datadog" style="width:100%;">}}
+
+## Configuration
+
+### Event types
+
+Audit Logs is configurable by event type. Enabled events are specific to actions that take place in the Datadog application. The following event types are supported for Audit Logs:
+
+- Access Management: Manage access, such as roles and users
+- Authentication: Successful and failed login attempts
+- Dashboard: Additions, deletions, and changes to dashboard resources
+- Log Management: Additions, deletions, and changes to log management resources
+- Monitor: Additions, deletions, and changes to monitor resources
+- Notebooks: Additions, deletions, and changes to notebook resources
+- Request: All web requests
+- Security Monitoring: Additions, deletions, and changes to security monitoring resources
+
+{{< img src="account_management/audit_logs/event-types.png" alt="Audit Logs event types setup in Datadog" style="width:70%;">}}
+
+### Archiving
+
+Archiving is an optional feature for Audit Logs. You can use archiving to write to an S3 bucket and have your SIEM system read events from it. Archiving is a double-write feature that has variable write times, but on average all events are written to the S3 bucket within 15 minutes of generation.
```

Comment:
It lets you write to three cloud storage systems currently: Amazon S3, Google Cloud Storage, and Azure Storage

Reply:
Interesting, will update this. Thanks!